plugins/inputs/snmp/snmp.go

```diff
@@ -105,7 +109,7 @@ type Snmp struct {
 	Community string
 
 	// Parameters for Version 2 & 3
-	MaxRepetitions uint
+	MaxRepetitions int
```

Comment:
Just FYI, if this was changed to avoid the cast on line 613, it'll change again when the gosnmp lib is updated, as it's now a `uint8`.